examples/sparseserver-ui/client/settings.py

```diff
@@ -0,0 +1,86 @@
+# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from pipelineclient import MultiPipelineClient
+
+
+class FeatureHandler:
+
+    """
+    Class with front-end streamlit content features.
+    """
+
+    tasks_desc = "Select task:"
+    tasks = [
+        "Question Answering",
+    ]
+
+    variants_desc = "Select model:"
+    variants = {
+        "12-Layer BERT, Sparse Quantized, 99% of Base Accuracy": MultiPipelineClient(
+            model="question_answering/12l_pruned80_quant"
+        ),
+        "6-Layer BERT, Sparse Quantized, 96% of Base Accuracy": MultiPipelineClient(
+            model="question_answering/quant6lagg96"
+        ),
+        "3-Layer BERT, Sparse Quantized, 89% of Base Accuracy": MultiPipelineClient(
+            model="question_answering/quant3lagg89"
+        ),
+        "12-Layer BERT Base, Not Sparsified 😢": MultiPipelineClient(
```

Comment:
can we put this at top/first so as you go down the list the models get faster and less accurate?